index.html

```diff
@@ -80,6 +80,11 @@ <h5 class="quote-author" id="quoteAuthorTag">- Steve Jobs</h5>
           <div class="fbshare">
             <i class="fa-brands fa-facebook fa-2xl" style="color: #0084ff;"></i>
           </div>
+          <div class="twtshare">
+            <a class="twitter-share-button" href="https://twitter.com/intent/tweet?text=Hello%20world">
+              <i class="fa-brands fa-twitter fa-2xl" style="color: #5f99fc;"></i>
```

Comment:
Just a suggestion. Update twitter logo with the X logo

```suggestion
              <i class="fa-brands fa-x-twitter fa-2xl" style="color: #5f99fc;"></i>
```